ui/app/AppLayouts/Profile/Sections/SyncHistoryModal.qml

```diff
@@ -0,0 +1,70 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import QtQuick.Layouts 1.13
+import "../../../../imports"
+import "../../../../shared"
+import "../../../../shared/status"
+
+ModalPopup {
+    id: popup
+    title: qsTr("Sync history for")
+    
+    Column {
+        id: column
+        anchors.top: parent.top
+        anchors.bottom: parent.bottom
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.rightMargin: Style.current.padding
+        anchors.leftMargin: Style.current.padding
+
+        spacing: 0
+
+        ButtonGroup { id: syncSettings }
+
+        StatusRadioButtonRow {
+            text: qsTr("One day")
+            buttonGroup: syncSettings
+            checked: profileModel.mailservers.DefaultSyncPeriod === Constants.fetchRangeLast24Hours
```

Comment:
Might not be necessary, but we could put `profileModel.mailservers.DefaultSyncPeriod` in a  property above so that it is fetched only once.